content/how-to/account-management/account-switch-feature/index.md

```diff
@@ -0,0 +1,29 @@
+---
+permalink: account-switch-feature
+audit_date: '2020-09-09'
+title: Account switch feature
+type: article
+created_date: '2020-09-09'
+created_by: Rose Morales
+last_modified_date: '2020-09-09'
+last_modified_by: Rose Morales
+product: Account Management
+product_url: account-management
+---
+
+You can choose the **Remember Me** option and up to 5 remembered users will appear on the user menu. This makes account switching for Dedicated and Managed Hosting seamless and easy.
+
+### Add user
+
+1. Login to [portal](login.rackspace.com).
+2. Select the **Remember Me** checkbox and the user profile is added to the **User** menu.
```

Comment:
2. Select the **Remember Me** checkbox.
    Your user profile appears on the **User** menu.